go.mod

```diff
@@ -228,6 +229,7 @@ require (
 	github.com/opencontainers/runtime-tools v0.9.1-0.20250523060157-0ea5ed0382a2 // indirect
 	github.com/opencontainers/selinux v1.12.0 // indirect
 	github.com/openshift/custom-resource-status v1.1.3-0.20220503160415-f2fdb4999d87 // indirect
+	github.com/pborman/uuid v1.2.0 // indirect
```

Comment:
This package does not seem to be maintained. We need to ensure everything we import can meet our requirements for FedRAMP Moderate and High Compliance. Is there a different package we can use? If not we should not be bringing in unmaintained upstreams and need to identify a different approach to the problem this solves.